apps/src/lab2/views/ExtraLinksModal.tsx

```diff
@@ -27,10 +34,26 @@ const ExtraLinksModal: React.FunctionComponent<ExtraLinksModalProps> = ({
   const [clonedLevelName, setClonedLevelName] = useState('');
   const [cloneError, setCloneError] = useState('');
   const [deleteError, setDeleteError] = useState('');
+  const [featuredProjectStatus, setFeaturedProjectStatus] = useState<
+    string | undefined
+  >('');
+
+  const channelId: string | undefined = useAppSelector(
+    state => state.lab.channel && state.lab.channel.id
+  );
+  const levelName = levelbuilderLinkData.level_name;
+  const isStandaloneProject =
```

Comment:
Sure - that makes sense. Will update.

Reply:
@sanchitmalhotra126 - just tagging since you wrote a comment related to this below - related to https://github.com/code-dot-org/code-dot-org/pull/59283#discussion_r1651330227.